pydicom/data/download.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2018-2020 pydicom authors. See LICENSE file for details.
```

Comment:
Happy to have it say 2020, but I chose that time span since the files were heavily based on the following:

* https://github.com/pymedphys/pymedphys/blob/master/pymedphys/_data/download.py
* https://github.com/pymedphys/pymedphys/blob/master/pymedphys/_utilities/filehash/core.py

Figured it would just be easier to relicence them under MIT so you don't need another license filling up your license file.